images/Dockerfile

```diff
@@ -5,7 +5,7 @@ ARG TARGETOS
 ARG TARGETARCH
 ARG RUNNER_VERSION
 ARG RUNNER_CONTAINER_HOOKS_VERSION=0.3.2
-ARG DOCKER_VERSION=20.10.23
+ARG DOCKER_VERSION=24.0.5
```

Comment:
i would suggest we go to `23.0.6` which matches the hosted runner image https://github.com/actions/runner-images/blob/main/images/linux/Ubuntu2204-Readme.md#tools

Reply:
@TingluoHuang, updated!